jooq-pg/src/main/resources/db/migration/ehr/V15__data_tables_add_parent_num.sql

```diff
@@ -16,137 +16,275 @@
  * limitations under the License.
  */
 
-ALTER TABLE comp_data
+
+-- Ensure clean state
+DROP FUNCTION IF EXISTS ext.mig_calc_nums(ext.mig_num_type[]);
+DROP FUNCTION IF EXISTS ext.mig_retrieve_nums_batch(text, text, text, text, text, text, integer);
+DROP PROCEDURE IF EXISTS ext.mig_num_columns(regclass, text, text, integer);
+DROP TYPE IF EXISTS ext.mig_num_type CASCADE;
```

Comment:
Move all the functions etc. to schema pg_temp.
Dropping mig_num_type will also drop the functions (but not the procedure).
Also, the CREATE OR REPLACE on the procedure should be sufficient.